### PR TITLE
Add metal bushes compatibility

### PR DESCRIPTION
**Pull request by nbrichau**

### CODE REVIEW

File: mod_metalbushesmod/plants/xp_bush_plant.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  "seed_drop_bonus": 0.0,
+  "products": {
+    "products": [
+      {
+        "min": 1,
+        "max": 1,
+        "chance": 0.85,
+        "required": true,
+        "type": "item",
+        "object": "minecraft:experience_bottle",
```
[the remark this answers is not in the document]
Comment (nbrichau) [reply mod_metalbushesmod/plants/xp_bush_plant.json:55]: No, they spawn experience orb entities but there is no way to summon an entity so I had to adapt.

[the remark this answers is not in the document]
Comment (InfinityRaider) [reply mod_metalbushesmod/plants/xp_bush_plant.json:55]: I'll create a callback for that.

[the remark this answers is not in the document]
Comment (InfinityRaider) [reply mod_metalbushesmod/plants/xp_bush_plant.json:55]: @nbrichau the experience callback has been implemented, you can remove the drops from the xp bush, and use the callback.

**Comment by InfinityRaider** [comment]
Thanks!

**Pull request merged**